pkg/cli/expose/expose.go

```diff
@@ -78,8 +78,10 @@ type ExposeOptions struct {
 }
 
 func NewExposeFlags(streams genericiooptions.IOStreams) *ExposeFlags {
+	flags := expose.NewExposeFlags(streams)
+	flags.PrintFlags = genericclioptions.NewPrintFlags("exposed").WithTypeSetter(exposeCmdScheme)
```

Comment:
Curiously enough, this is the only place that needed to use `exposeCmdScheme` for `oc expose` to work in the simple local tests I ran. However, for consistency sake I kept the use of `exposeCmdScheme` throughout this file.

Reply:
That is exactly what I suggested. Thank you.